scipy/stats/stats.py

```diff
@@ -944,8 +944,17 @@ def moment(a, moment=1, axis=0):
             # the input was 1D, so return a scalar instead of a rank-0 array
             return np.float64(0.0)
     else:
-        mn = np.expand_dims(np.mean(a, axis), axis)
-        s = np.power((a - mn), moment)
+        # Subtract mean along the axis, compute element-wise square
+        a_zero_mean = a - np.expand_dims(np.mean(a, axis), axis)
+        a_zero_mean_2 = a_zero_mean**2
+        
+        s = a_zero_mean_2.copy()
+        for k in range(1, mom // 2):
```

Comment:
Ah, yes of course. Silly mistake. I have fixed the code locally, would pushing update it here in the pull request as well?

runtests.py reports the following:

FAIL: test_skewtest_2D_WithMask (test_mstats_basic.TestCompareWithStats)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/stefan/ghproj/scipy/build/testenv/lib/python3.4/site-packages/scipy/stats/tests/test_mstats_basic.py", line 1013, in test_skewtest_2D_WithMask
    assert_equal(r[0][0],rm[0][0])
  File "/usr/lib/python3/dist-packages/numpy/ma/testutils.py", line 100, in assert_equal
    raise AssertionError(msg)
AssertionError: 
Items are not equal:
 ACTUAL: 0.38677410656235006
 DESIRED: 0.38677410656235239

should anything be done about that?